polybar-scripts/player-spotify/README.md

```diff
@@ -0,0 +1,25 @@
+# Script: player-spotify
+
+Displays current song and controls.
+
+![player-spotify](screenshots/1.png)
+
+
+## Dependencies
+
+- Spotify
+- dbus
+
+
+## Module
+
+```ini
+[module/spotify]
+type= custom/script
+exec = ~/.config/polybar/spotify.sh
+label =/ %{A1:dbus-send --print-reply --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.Previous:}<%{A} %{A1:dbus-send --print-reply --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.PlayPause:} %output% %{A} %{A1:dbus-send --print-reply --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.Next:}>%{A}
```

Comment:
We have a lot of code here in our config file. An idea how to reduce this?
You could use functions in your script and call the different functions with a script parameter.